docs/devguide/repl.rst

```diff
@@ -4,42 +4,90 @@
 Accessing the REPL
 ==================
 
-Accessing the REPL on the micro:bit requires:
+REPL (Read-Evaluate-Print-Loop) allows the micro:bit to read and evaluate code 
+in real-time as you write it.
 
-    * Using a serial communication program
-    * Determining the communication port identifier for the micro:bit
-    * Establishing communication with the correct settings for your computer
+Accessing the REPL on the micro:bit will require you to:
 
-If you are a Windows user you'll need to install the correct drivers. The
+    * Determine the communication port identifier for the micro:bit
+    * Use a program to establish communication with the device
+
+If you are a Windows user you'll need to install the correct drivers, the
 instructions for which are found here:
 
 https://developer.mbed.org/handbook/Windows-serial-configuration
 
-Serial communication
+
+Using a serial communication program
+------------------------------------
+
+The `Mu Editor <https://codewith.mu/en/tutorials/1.0/repl>`_ has built-in 
+support for REPL and even includes a real-time data plotter. Some other common 
+options are `picocom` and `screen`. You will need to install a program and 
+read the appropriate documentation to understand the basics of connecting to a 
+device.
+
+
+Determining the port
 --------------------
 
-To access the REPL, you need to select a program to use for serial communication.
-Some common options are `picocom` and `screen`. You will need to install
-program and understand the basics of connecting to a device.
+The micro:bit will have a port identifier (tty, usb) that can be used by the 
+computer for communicating. Before connecting to the micro:bit we must 
+determine the port identifier.
+
+**Windows**
+
+When you have installed the aforementioned drivers the micro:bit will appear in
+device-manager as a COM port.
+
+**Mac OS**
+
+Open Terminal and type ``ls /dev/cu.*`` to see a list of connected serial 
+devices; one of them will look like ``/dev/cu.usbmodem1422`` (the exact number 
+will depend on your computer).
+
+**Linux**
+
+In terminal, type ``dmesg | tail`` which will show which ``/dev`` node the 
+micro:bit was assigned (e.g. ``/dev/ttyUSB0``).
+
+
+Communicating with the micro:bit
+--------------------------------
+
+Once you have found the port identifier you can use a program to communicate 
```

Comment:
Can we change `... you can use a program to..` -> `... you can use a serial terminal program to ...`?
Makes it a bit more clear.